.tool/genheader.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017 The Linux Foundation
```

Comment:
might want to change the year here..

Reply:
hmmm, fair. It's fairly directly copied code, but I get it.

Reply:
yeah one way to handle that is to put a year range or just add one for this year..